runtime/src/main/scala/com/soundcloud/twinagle/ClientEndpointBuilder.scala

```diff
@@ -11,13 +11,15 @@ class ClientEndpointBuilder(httpClient: Service[Request, Response],
     extension(endpointMetadata).toFilter andThen
       new JsonClientFilter[Req, Resp](endpointMetadata.path) andThen
       new TwirpHttpClient andThen
+      new TracingFilter(endpointMetadata, isClient = true) andThen
```

Comment:
the filter needs to move up in the stack, right after the `extension()` line. As is is now, it only wraps the HTTP client, so it won't include any twirp-specific logic (in particular, it won't be able to record `TwinagleExceptions`, as mentioned below.

Reply:
addressed (by you)